src/intents.js

```diff
@@ -46,6 +46,14 @@ function injectService (url, element, intent, data) {
         return event.source.postMessage(data, event.origin)
       }
 
+      if (handshaken && event.data.type === `intent-${intent._id}:size`) {
+        ['width', 'height', 'maxWidth', 'maxHeight', 'minWidth', 'minHeight'].forEach(prop => {
```

Comment:
Be ware not to predict the future of the usage (as it could change very soon), it will be harder to maintain (as it could become very far from the target). That is not my advice, but Kent Beck's one: https://blog.ticabri.com/2015/07/31/comment-predire-le-futur-de-la-programmation/. My 2 cents :angel: